README.md

```diff
@@ -30,3 +30,44 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+ANSWER:
+
+Looking at the given function the recurrence relation can be derived as follow:
+
+$T(n) = 3T(n/3) + n^5$
+
+Starting off the base case where $n <= 1$ takes $O(1)$ time complexity. Next we can see that there are three recursive calls which represents the $3T$, and the $n/3$ is defined by recursive calls with n divided into 3 parts. We get the $O(n^5)$ from the nested loops where i is set to be < $n^2$, j is set to be < $n$, k is < $n^2$. In total it takes $O(n^5)$ time complexity.
+
+Solving for total time complexity:
+
+we know that T(1) = 1,
+
+$T(n/3) = 3(3T(n/9) + (n/3)^5) + n^5$
+
+$= 9T(n/9) + (n/3)^5 + n^5$
+
+$= 9(3T(n/27) + (n/9)^5) + (n/3)^5 + n^5$
+
+The above continues until the code uses recursion, we get
+
+$= 3^i(n/3^i) + \sum_{j=0}^{i-1}(3^j)(n/3^j)^5$
+
+$= 3^i(n/3^i) + n^5 \sum_{j=0}^{i-1}(3^j/3^5j)$
+
+The recursion stops when the array size becomes 1 meaning $n/3^i=1$
+
+if we solve for i we get $i=log_3(n)$
+
+substitute this in the above equation we get 
+
+$T(n) = 3^(log_3(n)) * T(n/3^(log_3(n))) + n^5 \sum_{j=0}^{log_3(n)-1}(3^j/3^5j)$
+
+using logarithmic functions we know that $3^(log_3(n)) = n$
+
+$T(n) = n * T(1) + n^5$
```

Comment:
And what about the sum that you had above?

Reply:
I changed my file

Reply:
The sum still just disappears? Where did it go?

Reply:
I am so sorry I thought I included it and completely ignored to check as it equates to a number that is less than 1 which can be ignored asymptotically. I have updated my file to reflect this answer along with the website that helped with the geometric series formula with examples.